solutions/usaco-104.mdx

```diff
@@ -2,15 +2,15 @@
 id: usaco-104
 source: USACO Bronze 2012 January
 title: Haybale Stacking
-author: Óscar Garries
+author: Óscar Garries, Brad Ma
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_stacking.html)
 
-<LanguageSection>
+</LanguageSection>
```

Comment:
```suggestion
<LanguageSection>
```
opening tags don't have `/`, closing tags do

Reply:
rip saw a suggestion for that so I just added the slash oops